### DISCUSSION TITLE
How to maintain same _rsc key for each link in all the pages

### DISCUSSION THREAD

**Discussion by vinothweb** [vercel/next.js#63599] [Help]
### Summary

I was wondering how to maintain same _rsc key for links across the pages. Below are example,

```
In page -  **example.com/about** - the key for that about **page will be "HYFGB"**, example link below
example.com/contact?_rsc=HYFGB

In Page - **example.com/story** -  the key for that story **page will be "AKYGV"**, example link below
example.com/contact?_rsc=AKYGV
```
In the above example the _rsc key of contact page is different in about and story pages. This leads to fetch the rsc pages of same link multiple times in different pages. 

I am expecting to use same **_rsc=XXXX** key across all the pages for the links.

**Comment by 0xD3ADBE3F** [comment]
Somewhat related is discussed over here #59167
